tests/prefetch_related/tests.py

```diff
@@ -1510,3 +1510,45 @@ def test_retrieves_results_from_prefetched_objects_cache(self):
         with self.assertNumQueries(4):
             # AuthorWithAge -> Author -> FavoriteAuthors, Book
             self.assertQuerysetEqual(authors, ['<AuthorWithAge: Rousseau>', '<AuthorWithAge: Voltaire>'])
+
+    def test_related_manager_remove_prefetched_objects(self):
+        """
+        Test RelatedManager.remove_prefetched_objects method
+        """
+        books = Book.objects.prefetch_related('first_time_authors')
+        book1 = books.get(title='Les confessions Volume I')
+        book2 = books.get(title='Candide')
+
+        # Update the related object - author1. So book1 has no first-time author now, but still has one in cache
+        author1 = book1.authors.first()
+        author1.first_book = book2
+        author1.save()
+
+        self.assertQuerysetEqual(
```

Comment:
Use `assertCountEqual()` because it's simpler.

Reply:
Changed to `assertSequenceEqual`